realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -850,7 +850,8 @@ private boolean containsDot(String s) {
      * Condition that value of field contains the specified substring
      * @param fieldName  The field to compare
      * @param value      The substring
-     * @param caseSensitive if true, substring matching is case sensitive
+     * @param caseSensitive If true, substring matching is case sensitive. Setting this to false only works for ASCII
```

Comment:
No, I forgot the other methods. beginsWith(), endsWith(), equalTo(). They are being added now